ros2/test/diagnostics/tests.py

```diff
@@ -21,6 +21,10 @@
 import launch_testing.asserts
 import launch_testing.markers
 import rclpy
+from python.runfiles import Runfiles
+
+AGGREGATOR_NODE_PATH = Runfiles.Create().Rlocation(
+    'ros2_diagnostics/aggregator_node')
```

Comment:
This is interesting. I am curious what reads the path now. Can you paste the path here, please? Just for the future reference.

Reply:
```
# Workspace
working dir:  /home/laurenz/.cache/bazel/_bazel_laurenz/560f8735b75dc5afa81db7c5a8d429bc/sandbox/linux-sandbox/169/execroot/com_github_mvukov_rules_ros2/bazel-out/k8-fastbuild/bin/ros2/test/diagnostics/diagnostic_heartbeat_test.runfiles/com_github_mvukov_rules_ros2
runfile path: /home/laurenz/.cache/bazel/_bazel_laurenz/560f8735b75dc5afa81db7c5a8d429bc/sandbox/linux-sandbox/196/execroot/com_github_mvukov_rules_ros2/bazel-out/k8-fastbuild/bin/ros2/test/diagnostics/diagnostic_heartbeat_test.runfiles/ros2_diagnostics/aggregator_node

# Bzlmod (on the bzlmod branch)
working dir:  /home/laurenz/.cache/bazel/_bazel_laurenz/560f8735b75dc5afa81db7c5a8d429bc/sandbox/linux-sandbox/166/execroot/_main/bazel-out/k8-fastbuild/bin/ros2/test/diagnostics/diagnostic_heartbeat_test.runfiles/_main
runfile path: /home/laurenz/.cache/bazel/_bazel_laurenz/560f8735b75dc5afa81db7c5a8d429bc/sandbox/linux-sandbox/162/execroot/_main/bazel-out/k8-fastbuild/bin/ros2/test/diagnostics/diagnostic_heartbeat_test.runfiles/_main~non_module_deps~ros2_diagnostics/aggregator_node
```